resources/fixtures/trade_aggregation.json

```diff
@@ -0,0 +1,27 @@
+{
+  "timestamp": 1517522400000,
+  "trade_count": 26,
+  "base_volume": "27575.0201596",
+  "counter_volume": "5085.6410385",
+  "avg": "0.1844293",
+  "high": "0.1915709",
+  "high_r": {
```

Comment:
These ratios didn't get returned in the curl:

```
$ curl 'https://horizon.stellar.org/trade_aggregations?base_asset_type=native&counter_asset_code=SLT&counter_asset_issuer=GCKA6K5PCQ6PNF5RQBF7PQDJWRHO6UOGFMRLK3DYHDOI244V47XKQ4GP&counter_asset_type=credit_alphanum4&limit=200&order=asc&resolution=3600000&start_time=1517521726000&end_time=1517532526000'
{
  "_links": {
    "self": {
      "href": "https://horizon.stellar.org/trade_aggregations?base_asset_type=native\u0026counter_asset_code=SLT\u0026counter_asset_issuer=GCKA6K5PCQ6PNF5RQBF7PQDJWRHO6UOGFMRLK3DYHDOI244V47XKQ4GP\u0026counter_asset_type=credit_alphanum4\u0026limit=200\u0026order=asc\u0026resolution=3600000\u0026start_time=1517521726000\u0026end_time=1517532526000"
    },
    "next": {
      "href": "https://horizon.stellar.org/trade_aggregations?base_asset_type=native\u0026counter_asset_code=SLT\u0026counter_asset_issuer=GCKA6K5PCQ6PNF5RQBF7PQDJWRHO6UOGFMRLK3DYHDOI244V47XKQ4GP\u0026counter_asset_type=credit_alphanum4\u0026end_time=1517532526000\u0026limit=200\u0026order=asc\u0026resolution=3600000\u0026start_time=1517529600000"
    },
    "prev": {
      "href": ""
    }
  },
  "_embedded": {
    "records": [
      {
        "timestamp": 1517522400000,
        "trade_count": 26,
        "base_volume": "27575.0201596",
        "counter_volume": "5085.6410385",
        "avg": "0.1799230",
        "high": "0.1915709",
        "low": "0.1506024",
        "open": "0.1724138",
        "close": "0.1506024"
      },
      {
        "timestamp": 1517526000000,
        "trade_count": 15,
        "base_volume": "3913.8224543",
        "counter_volume": "719.4993608",
        "avg": "0.1775659",
        "high": "0.1960784",
        "low": "0.1506024",
        "open": "0.1960784",
        "close": "0.1515152"
      }
    ]
  }
}
```

Reply:
Should we delete them from the fixture?  I tried to click through on the stellar site where it says "edit this doc on github" but couldn't find the repo.

Reply:
meh, it doesn't hurt having it.